src/plugins/compile_worker.ts

```diff
@@ -98,7 +99,7 @@ process.on("message", async m => {
 			.get("https://ethereum.github.io/solc-bin/bin/list.json")
```

Comment:
This list is not updated anymore.
you can use https://solc-bin.ethereum.org/wasm/list.json or https://solc-bin.ethereum.org/bin/list.json